HisashiburiDana.Api/Controllers/AuthenticationController.cs

```diff
@@ -76,6 +78,79 @@ public async Task<IActionResult> ReIssueToken(RefreshTokenRequest request)
             }
             return Ok(loginResponse);
         }
-        
+
+
+        /// <summary>
+        /// Forgot Password
+        /// </summary>
+        /// <param name="email"></param>
+        /// <returns></returns>
+        [Route("forgotpassword")]
+        [HttpPost]
+        [Produces(typeof(GeneralResponseWrapper<bool>))]
+        public async Task<IActionResult> ForgotPassword(string email)
+        {
```

Comment:
Normally when you forget password. You aren't logged in. So the email from the claims should be null